volatility3/framework/plugins/windows/mftscan.py

```diff
@@ -0,0 +1,331 @@
+# This file is Copyright 2022 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import logging
+
+from struct import unpack
+from typing import Dict
+
+from volatility3.framework import constants, renderers, interfaces
+from volatility3.framework.configuration import requirements
+from volatility3.framework.exceptions import PagedInvalidAddressException
```

Comment:
I'd prefer to import `exceptions` rather than `PagedInvalidAddressException` to avoid people accidentally deciding to import it from here instead of where it lives.